Client/TeamTalkClassic/TeamTalkBase.h

```diff
@@ -306,7 +306,9 @@ enum SoundEvent
     SOUNDEVENT_NONE = 0,
     SOUNDEVENT_USER_JOIN                    = 0x00000001,
     SOUNDEVENT_USER_LEFT                    = 0x00000002,
+    SOUNDEVENT_USER_LOGGED_IN               = 0x00000003,

```

Comment:
This is a bitmask. SOUNDEVENT_USER_LOGGED_IN would be the same as (SOUNDEVENT_USER_LEFT | SOUNDEVENT_USER_JOIN). I'll fix it.